velox/dwio/dwrf/common/Config.h

```diff
@@ -78,6 +78,48 @@ class Config : public config::ConfigBase {
   static Entry<uint64_t> RAW_DATA_SIZE_PER_BATCH;
   static Entry<bool> MAP_STATISTICS;
 
+  /// Maximum stripe size in orc writer.
+  static constexpr const char* kOrcWriterMaxStripeSize =
+      "hive.orc.writer.stripe-max-size";
```

Comment:
Do we need hive prefix in the config? If it is hive specific, then shall we pass connector id to processConfig API (this might cause the dependency from file format to connector config) or does hive data sink should convert hive specific config to the corresponding format ones?

Reply:
@xiaoxmeng We have the same dilemma for others like storage adapters (s3, gcs) config as well. eg. We use `hive.s3.` for S3.
We should discuss this and modify all the configs in a separate PR.

Reply:
In a separate PR, I think we can remove the `hive` prefix for all and modify the config in the HiveConnector before passing to file formats, storage-adapters.